services/QuillLMS/app/views/layouts/canvas_integration/lti.html.erb

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+  <body>
+    <div style='background-color:green'>
```

Comment:
This might be better in a subsequent PR, but we probably want to add minimal inline styling on this page to match the general Quill site (I'm suggesting inline to keep the page load quick, but maybe I'm over optimizing) 

I think we'd want something like a minimal version of our current [login page](https://www.quill.org/session/new) , e.g. minimal header with correct green, beige background with a white box for the content and white button, minimal footer with just the copyright info.

![Screenshot 2023-06-09 at 5 05 20 PM](https://github.com/empirical-org/Empirical-Core/assets/1304933/3917ffd0-26c0-4820-bb92-04339ac7b529)
![Screenshot 2023-06-09 at 5 12 44 PM](https://github.com/empirical-org/Empirical-Core/assets/1304933/0d64a3ac-9ee8-48c2-8e15-35c038f1900e)